modules/ballerina-builtin/src/main/ballerina/ballerina/data/sql/natives.bal

```diff
@@ -3,11 +3,11 @@ package ballerina.data.sql;
 @Description { value: "Parameter struct represents a query parameter for the SQL queries specified in connector actions"}
 @Field {value:"sqlType: The data type of the corresponding SQL parameter"}
 @Field {value:"value: Value of paramter pass into the SQL query"}
-@Field {value:"direction: Direction of the SQL Parameter 0 - IN, 1- OUT, 2 - INOUT"}
+@Field {value:"direction: Direction of the SQL Parameter IN, OUT, or INOUT"}
 public struct Parameter {
-	string sqlType;
+	data sqlType;
 	any value;
-	int direction;
+	dir direction;
```

Comment:
Shall we reword the data type? "dir" might confuse with "directory" (coming from unix world).

Reply:
Changed the newly added enum types as "Direction", "Type" and "DB". Thanks.